Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11698,6 +11698,22 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Sends the specified message to the player.",
 				},
+				SendMessageRaw =
+				{
+					Params =
+					{
+						{
+							Name = "Json",
+							Type = "string",
+						},
+						{
+							Name = "eChatType",
+							Type = "number",
+							IsOptional = true,
+						}
+					},
+					Notes = "Sends the specified json string to the player. The optional value eChatType (default ctChatBox) can be ctChatBox, ctSystem or ctAboveActionBar. You can use {{cJson|cJson}} to build a json string.",
```

Comment:
You don't need the second `|cJson` in the link. The syntax is `{{url|linktext}}` or `{{url}}` (if the link text is the same as url; url can be a class name. So you can use `{{cJson}}` or `{{cJson|the built in JSON serializer}}` etc.